.storybook/manager-head.html

```diff
@@ -1 +1,3 @@
-<script>document.title = "UI Kit";</script>
+<script>
+  document.title = "UI Kit"; window.STORYBOOK_GA_ID = "UA-38285631-31";
```

Comment:
Wondering if this is the best place to set the STORYBOOK_GA_ID.

Reply:
Could work yeah 👌